docs/user/vulnerabilities/vulnerability/cwe/name-usage.en.md

```diff
@@ -1 +1,7 @@
 # Name - Usage
+
+The full name as given in the specification.
+
+*Examples:*
+
+* CWE-79
```

Comment:
That example is incorrect: It should list the full name =>
CWE-79: `<full name here>`